apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -0,0 +1,1124 @@
+                        (o) => o.name === newVal
+                      );
+
+                      const writeIn = official
+                        ? undefined
+                        : writeInCandidates?.find((w) => w.name === newVal);
+
+                      let newWriteInState: WriteInOptionState;
+                      if (official) {
+                        newWriteInState = {
+                          type: 'existing-official',
+                          ...official,
+                        };
+                      } else if (writeIn) {
+                        newWriteInState = {
+                          type: 'existing-write-in',
+                          ...writeIn,
+                        };
+                      } else {
+                        newWriteInState = { type: 'new', name: newVal };
+                      }
+                      updateWriteInState(optionId, newWriteInState);
+                      if (!isSelected) {
+                        updateVote(optionId, true);
+                      }
+                    }}
+                    toggleVote={() => {
+                      if (isSelected) {
+                        if (isFocused) {
+                          setFocusedOptionId('');
+                        }
+                        updateVote(optionId, false);
+                        updateWriteInState(optionId, { type: 'invalid' });
+                      } else {
+                        updateVote(optionId, true);
+                        updateWriteInState(optionId, { type: 'pending' });
+                      }
+                    }}
+                    officialCandidateNames={officialCandidates
+                      .filter(
+                        (c) =>
+                          !selectedCandidateNames.includes(c.name) ||
+                          (isExistingCandidate(writeInEntry) &&
+                            writeInEntry.name === c.name)
+                      )
+                      .map((c) => c.name)}
+                    writeInCandidateNames={writeInCandidates
+                      .filter(
+                        (c) =>
+                          !selectedCandidateNames.includes(c.name) ||
+                          (isExistingCandidate(writeInEntry) &&
+                            writeInEntry.name === c.name)
+                      )
+                      .map((c) => c.name)}
+                    caption={renderCandidateButtonCaption({
+                      originalVote,
+                      currentVote: isSelected,
+                      isWriteIn: true,
+                      writeInRecord,
+                      writeInState: writeInEntry,
+                    })}
+                  />
+                );
+              })}
+            </CandidateButtonList>
+          )}
+          <BallotFooter>
+            <BallotMetadata>
+              <SmallText>
+                {scrollIndex + 1} of {numBallots}
```

Comment:
Ditto on count formatting, and here it might matter.
```suggestion
                {format.count(scrollIndex + 1)} of {format.count(numBallots ?? 0)}
```